src/components/MerchantsPage/styles.module.scss

```diff
@@ -145,8 +145,17 @@
 }
 
 .flyerContainer {
-  margin: 3vh 7vw;
+  $flyer-margin: 3rem;
+
+  margin-top: $flyer-margin;
+  margin-bottom: $flyer-margin;
+  font-size: 120%;
```

Comment:
Why is the font size a percent?

Reply:
The text in the flyer msg seems to be slightly larger than the normal text; is there any text component/mechanism you guys are using to keep font sizes consistent?